spec/OpenLineage.json

```diff
@@ -0,0 +1,425 @@
+              "type": "string",
+              "format": "date-time",
+              "example": "2020-12-17T03:00:00.000Z"
+            },
+            "nominalEndTime": {
+              "description": "An [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) timestamp representing the nominal end time (excluded) of the run. (Should be the nominal start time of the next run)",
+              "type": "string",
+              "format": "date-time",
+              "example": "2020-12-17T04:00:00.000Z"
+            }
+          },
+          "required": [
+            "nominalStartTime"
+          ]
+        }
+      ],
+      "type": "object"
+    },
+    "ParentRunFacet": {
+      "description": "the id of the parent run and job, iff this run was spawn from an other run (for example, the Dag run scheduling its tasks)",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseFacet"
+        },
+        {
+          "type": "object",
+          "properties": {
+            "run": {
+              "type": "object",
+              "properties": {
+                "runId": {
+                  "type": "string",
+                  "format": "uuid"
+                }
+              },
+              "required": [
+                "runId"
+              ]
+            },
+            "job": {
+              "type": "object",
+              "properties": {
+                "namespace": {
+                  "description": "The namespace containing that job",
+                  "type": "string",
+                  "example": "my-scheduler-namespace"
+                },
+                "name": {
+                  "description": "The unique name for that job within that namespace",
+                  "type": "string",
+                  "example": "myjob.mytask"
+                }
+              },
+              "required": [
+                "namespace",
+                "name"
+              ]
+            }
+          },
+          "required": [
+            "run",
+            "job"
+          ]
+        }
+      ],
+      "type": "object"
+    },
+
+    "DocumentationJobFacet": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseFacet"
+        },
+        {
+          "type": "object",
+          "properties": {
+            "description": {
+              "description": "The description of the job.",
+              "type": "string"
+            }
+          },
+          "required": [
+            "description"
+          ]
+        }
+      ],
+      "type": "object"
+    },
+    "SourceCodeLocationJobFacet": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseFacet"
+        },
+        {
+          "type": "object",
+          "properties": {
+            "type": {
+              "type": "string",
+              "example": "git"
+            },
+            "url": {
+              "type": "string",
+              "format": "uri"
```

Comment:
It is a bit confusing to me if this should be a url or uri.

Reply:
a URL is a URI, so this is correct? https://tools.ietf.org/html/rfc3986#section-1.1.3
built-in formats have uri only. https://json-schema.org/understanding-json-schema/reference/string.html#resource-identifiers